imperium/models/spacecraft.py

```diff
@@ -64,7 +66,7 @@ def __init__(self, hull_tonnage):
             item_tonnage = data.get(key).get("tonnage")
             if self.tonnage <= item_tonnage and (item_tonnage - self.tonnage) < 100:
                 self.hull_designation = key
-                self.cost_hull = data.get(key).get("cost")
+                self.cost_hull = self.base_cost_hull = data.get(key).get("cost")
```

Comment:
Is this double equal intentional?

Reply:
Yes this is, this is to get the inherent hull cost for determination of streamlined and distributed calculations.

A good change for this would be to have a state/enum/whathaveyou for use in the implicit getter calculations